src/Extension/CoreExtension.php

```diff
@@ -260,6 +260,8 @@ public function getTests(): array
             new TwigTest('constant', null, ['node_class' => ConstantTest::class]),
             new TwigTest('empty', [self::class, 'testEmpty']),
             new TwigTest('iterable', 'is_iterable'),
+            new TwigTest('sequence', [self::class, 'testSequence']),
+            new TwigTest('mapping', [self::class, 'testMapping']),
```

Comment:
I've just checked Twig docs and code, and we are using "sequence" and "hash".
So, we have 2 options here:
 * Rename "mapping" to "hash" to be consistent with Twig
 * Keep "mapping" here for consistency with Jinja and rename "hash" to "mapping" in current docs and code (that would not break anything).

I'd go with consistency with Jinja here.

What do others think? @stof @GromNaN @xabbuh

Reply:
+1 for consistency with Jinja.

Reply:
I'd also go for consistency with Jinja 👍

Reply:
"Mapping" is already used in some pages of the documentation. Examples:

https://twig.symfony.com/doc/3.x/filters/map.html
"The map filter applies an arrow function to the elements of a sequence or a mapping."

https://twig.symfony.com/doc/3.x/filters/first.html
"The first filter returns the first "element" of a sequence, a mapping, or a string"